src/relay/backend/aot_executor_codegen.cc

```diff
@@ -347,18 +347,30 @@ class AOTExecutorCodegen : public MixedModeVisitor {
     }
 
     GlobalVar global_var = call_lowered_props.lowered_func;
+    tir::Var empty_var("no_device_context", DataType::Handle());
     bool has_c_device_api_context = device_contexts_.count(global_var) != 0;
+    bool use_cpacked_api = !use_unpacked_api_;
+
     if (has_c_device_api_context) {
```

Comment:
Let's comment these three semi-implicit calling conventions here, including a one-sentence summary of where the extra args show up in the runtime.